pinot-common/src/main/java/org/apache/pinot/common/partition/ReplicaGroupPartitionAssignmentGenerator.java

```diff
@@ -82,7 +86,7 @@ public void writeReplicaGroupPartitionAssignment(ReplicaGroupPartitionAssignment
    * @return Replica group partition assignment from the given configuration
    */
   public ReplicaGroupPartitionAssignment buildReplicaGroupPartitionAssignment(String tableNameWithType,
-      TableConfig tableConfig, List<String> servers) {
+      TableConfig tableConfig, int numReplicas, List<String> servers) {
```

Comment:
I couldn't move ReplicationUtils to pinot-common, because it needs to use StreamConfig which is in pinot-core. This was the only way I could make it work. Open to suggestions..

Reply:
Isn't `StreamConifg` part of `TableConfig`? I think it makes more sense to put `StreamConfig` to `pinot-common` along with other configs?